### PR TITLE
Fix schedule E definition

### PR DESCRIPTION
**Pull request by ccostino**
This changeset addresses a few issues we uncovered while testing and validating the schedule E data updates we recently made for this release.  The form 5 filings had been lost and it did not resolve the missing 24/48 hour notices either.

/cc @jontours and @LindsayYoung

### CODE REVIEW

**Comment by jontours** [comment]
I have a change coming that will make this much simpler.  Standby!

**Comment by ccostino** [comment]
Awesome, thank you!

**Comment by ccostino** [comment]
@jontours or @LindsayYoung, this is now ready for review!

File: data/sql_setup/prepare_schedule_e.sql [hunk 1]
```diff
@@ -20,8 +20,54 @@ select *,
     now() as pg_date
 from fec_sched_e_notice_vw;
 
-update ofec_sched_e_tmp
```
Comment (jontours) [inline data/sql_setup/prepare_schedule_e.sql]: Woops...I think when you resolved the conflicts these two lines got deleted.  I cann add these back in.

**Pull request merged**